### PR TITLE
Fix bugs on subscription renewal

### PR DESCRIPTION
**Pull request by NoahCarnahan**
http://manage.dimagi.com/default.asp?210840
http://manage.dimagi.com/default.asp?210841

Fixes two small bugs with the `/a/<domain>/settings/project/subscription/renew/confirm/` page that were introduced in https://github.com/dimagi/commcare-hq/pull/9837.

@nickpell 
cc @snopoke

### CODE REVIEW

File: corehq/apps/domain/views.py [hunk 1]
```diff
@@ -1644,7 +1644,7 @@ class ConfirmSubscriptionRenewalView(DomainAccountingSettings, AsyncHandlerMixin
     @property
     @memoized
     def next_plan_version(self):
-        new_edition = self.request.POST.get('plan_edition').title()
+        new_edition = self.request.POST.get('plan_edition', "").title()
```
Comment (nickpell) [inline corehq/apps/domain/views.py]: do you know when ```POST``` won't have ```plan_edition```?

Comment (NoahCarnahan) [inline corehq/apps/domain/views.py]: It happens when you do a GET on the url that the POST is failing on. I think it's just a consequence of the user trying to refresh that page after it fails. Maybe we should disallow GETs on that view though?

Comment (nickpell) [inline corehq/apps/domain/views.py]: >Maybe we should disallow GETs on that view though?

Sounds like a good idea

Comment (NoahCarnahan) [inline corehq/apps/domain/views.py]: updated

**Comment by nickpell** [comment]
looks good to me!

**Pull request merged**